_ext/pipeline.rb

```diff
@@ -1,6 +1,7 @@
 require 'compass'
 require 'zurb-foundation'
 require 'awestruct_ext'
+require 'asciidoctor_ext'
```

Comment:
I think we should put this patch in awestruct_ext instead of asciidoctor_ext. No need for the extra file right now.